File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    s = list(map(str,s.split(' ')))
    answer = []
    
    for i in range(len(s)):
        tmp = ''
        for j in range(len(s[i])):
            if j==0:
                tmp += s[i][j].upper()
            else:
                tmp += s[i][j].lower()
        answer.append(tmp)
    
        
    
    return ' '.join(answer)

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.cpp
#include <string>
#include <cctype>
using namespace std;

string solution(string s) {
    string answer = "";
    bool flag = true; 
    
    for (char c : s) {
        if (c == ' ') {
            answer += c;
            flag = true;
        }
        else{
            if (flag) {
                answer += toupper(c);
                flag = false;
            }
            else {
                answer += tolower(c);
            }
        }
        
    }

    return answer;
}
